apps/public-docsite/src/SiteDefinition/SiteDefinition.tsx

```diff
@@ -50,6 +50,7 @@ export const SiteDefinition: ISiteDefinition<Platforms> = {
     { from: /#\/components/, to: '#/controls/web' },
     { from: '#/styles/animation', to: '#/styles/web/motion' },
     { from: '#/styles/brand-icons', to: '#/styles/web/office-brand-icons' },
+    { from: '#/styles/web/office-brand-icons', to: '#/styles/web/m365-brand-icons' },
```

Comment:
Even though line 52 connects with line 53, it is needed in case external links point to `#/styles/web/m365-brand-icons`